src/tori/models.rs

```diff
@@ -3,174 +3,109 @@ use serde::Deserialize;
 
 use crate::vahti::VahtiItem;
 
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriAccount {
-    code: String,
-    label: String,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriCategory {
-    code: String,
-    label: String,
-    name: String,
-    path_en: String,
-    parent: String,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriImage {
-    base_url: String,
-    media_id: String,
-    path: String,
-    width: i64,
-    height: i64,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriListPrice {
-    #[serde(default)]
-    currency: String,
-    price_value: i64,
-    label: String,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriMcSettings {
-    use_form: bool,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriListType {
-    code: String,
-    label: String,
-}
-
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriUserAccount {
-    name: String,
-    created: String,
+#[derive(Deserialize, Debug, Clone)]
+pub struct ToriImage {
+    pub url: String,
+    pub path: String,
+    pub height: i64,
+    pub width: i64,
+    pub aspect_ratio: f32,
 }
 
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriUser {
-    account: ToriUserAccount,
-    uuid: String,
+#[derive(Deserialize, Debug, Clone)]
+pub struct ToriCoordinates {
+    pub lon: f32,
+    pub lat: f32,
 }
 
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriPivo {
-    enabled: bool,
+#[derive(Deserialize, Debug, Clone)]
+pub struct ToriLabel {
+    pub id: String,
+    pub text: String,
+    pub r#type: String,
 }
 
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriListTime {
-    label: String,
-    value: i64,
+#[derive(Deserialize, Debug, Clone)]
+pub struct ToriPrice {
+    pub amount: i64,
+    pub currency_code: String,
+    pub price_unit: String,
 }
 
-#[derive(Deserialize, Debug, Clone, Default)]
-struct ToriLocation {
-    code: String,
-    key: String,
-    label: String,
-    #[serde(default)]
-    locations: Vec<ToriLocation>,
+/// An item returned by the search request
+#[derive(Deserialize, Debug, Clone)]
+pub struct ToriItem {
+    /// Type of the item for Torimies always "bap"
+    //pub r#type: String,
```

Comment:
Prolly get rid of this, if its unused.

If it is, it is much better to use `#[serde(rename = "type")]` and name the field something less keywordy